content/en/docs/concepts/storage/volumes.md

```diff
@@ -919,7 +927,7 @@ A container using a projected volume source as a [`subPath`](#using-subpath) vol
 receive updates for those volume sources.
 {{< /note >}}
 
-### quobyte
+### quobyte (deprecated)
```

Comment:
```suggestion
### quobyte (deprecated) {#quobyte}
```

This avoids breaking existing hyperlinks.

Reply:
Done